proxies/nodejs/handlers/location.ts

```diff
@@ -0,0 +1,95 @@
+import Koa from "koa";
+import { DVCClient, DVCUser } from "@devcycle/nodejs-server-sdk";
+
+type LocationRequestBody = {
+  command: string;
+  params: string; //[{value: string | number | boolean} | {location: string} | {callbackUrl: string}] HTTP request comes in as string
+  isAsync: boolean;
+};
+
+export const handleLocation = async (
+  ctx: Koa.ParameterizedContext,
+  data: {
+    clients: { [key: string]: DVCClient };
+    users: { [key: string]: DVCUser };
+  }
+) => {
+  const body = ctx.request.body as LocationRequestBody;
+  const urlParts = ctx.request.url.split("/");
+  console.log(urlParts);
+  let entity = getEntityFromLocation(ctx.request.url, data);
+  console.log("entity", entity);
+
+  if (entity === undefined) {
+    ctx.status = 400;
+    ctx.body = {
+      errorCode: 400,
+      errorMessage: "Invalid request: missing entity",
+    };
+  } else if (body.command === undefined) {
+    ctx.status = 400;
+    ctx.body = {
+      errorCode: 400,
+      errorMessage: "Invalid request: missing command",
+    };
+  } else {
+    try {
+      const command = body.command;
+      const params: any | string | boolean | number = parseParams(
+        JSON.parse(body.params),
+        data
+      );
+
+      // TODO: handle async commands
+      // TODO: handle callback before invoking command
+      const resultData = entity[command](...params);
+
+      ctx.status = 200;
+      ctx.body = {
+        entityType: typeof resultData, // assuming this is the type of the entity for returned data
```

Comment:
`entityType` has to be one `Variable`, `User`, `Feature`, or `Object`. The reason being is that the class names vary across the SDKs  (DVCVariable Vs. Variable) so we need to have them agree on the same entity types. This way the test harness won't have to worry about the SDKs individually

Reply:
I was waiting on the `EntityTypes` are added to my code to ensure that that can be used to find the correct corresponding type for the saved objects. 

For the the function outputs, we may need to find a way to map the types to the command types to ensure we are not getting `any` response resulting in `Object` type